organisms/organisation-unit-tree/new-features/select-descendants.feature

```diff
@@ -0,0 +1,16 @@
+Feature: Explicitly select individual descendant org units inside another org unit
+
+Background:
+  Given an org unit tree with root org unit 'Sierra Leone'
+  And descendants inside 'Sierra Leone' with multiple groups and levels
+
+  Scenario: Selecting all org units of a certain group or level inside another org unit
+    Given the user has opened the context menu for org unit 'Sierra Leone'
+    When the user selects a group or level type from the 'Select children >' submenu
+    Then all org units of that group or level inside 'Sierra Leone' are explicitly selected
```

Comment:
No need for the word `explicitly` in this sentence (Using the phrase `Implicit` selection in this case sounds illogical)?

Reply:
True. I suppose I'm trying to make a distinction here, which in my mind goes like: 
- Explicit selection: a ticked checkbox. Selected on FE
- Group/Level selection: an indication of a selection that will be made BE on runtime

Is there a better way of making this distinction - or perhaps it doesn't need to be made here?

Reply:
Tricky! The only variants I can come up with right now are:

`Then all org units of that group or level inside 'Sierra Leone' are selected individually`
`Then all org units of that group or level inside 'Sierra Leone' are simply selected`

The others could be "computationally selected" or "selecton derived from relative indicator/base", I guess?

One note: Maybe we should remove "group" from the `Then` sentence as it means you could select units in deeper levels as well (at least `groups or levels` seems to indicate that possibility)

Reply:
Thanks for the suggestions. I went for 'individually selected', I think that gets the point across.

> One note: Maybe we should remove "group" from the Then sentence as it means you could select units in deeper levels as well (at least groups or levels seems to indicate that possibility)
I've reworded to say the 'chosen group or level' - indicating it's based on what the user selected in the `When` step. Does that make more sense? I'm not sure I fully understand the point about deeper levels.